server/index.js

```diff
@@ -9,13 +9,16 @@ const app = new Koa();
 const router = new Router();
 
 const port = process.env.API_PORT || 3000;
+const db = process.env.NODE_ENV === 'test'
+  ? process.env.DB_TEST
```

Comment:
I like the naming for consistency but can you change the `.env.example` and also the `questions.test.js` mongoose connect call as it now throws errors with your environment variable.